### ISSUE TITLE
How to generate motion masks?

### ISSUE THREAD

**Issue by rkakash59** [fangli333/SC-4DGS#1]
Nice work, Pose free dynamic scene gaussian splatting was something I was waiting on. 
Congratulations on being the first one to do it.

I wanted to know how motion masks are generated. Because if it cannot be generated easily for most scenes then this will be a big limitation.

And also any estimate on code release time.

**Comment by fangli333** [comment]
Hi,

Thank you for your comments. For now, we use the G.T. motion mask. As mentioned in our paper, we take developing an efficient way to tackle the motion mask problem as our future goal. Please continue following our research. 

The way of generating motion masks in [RoDynRF](https://github.com/facebookresearch/robust-dynrf) might help you. However, we also tried their way, and we found that their approach always fails, which is also the motivation for our future goal.

For the estimated time of code release, we will release the code soon. Will make comments in this issue once released.

Best,
Fang

**Comment by rkakash59** [comment]
Thank you for your reply. 
Looking forward for your future work.
